e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -365,7 +368,7 @@
                 "value": "<TestResultCoordinator.ConsumerGroupId>"
               },
               "optimizeForPerformance": {
-                "value": "<TestResultCoordinator.OptimizeForPerformance>"
```

Comment:
I have made `OptimizeForPerformance` configuration apply consistently in all places. So I added it for `twinTester` also.